src/wallet/db.h

```diff
@@ -55,8 +55,7 @@ class CDBEnv
     enum VerifyResult { VERIFY_OK,
                         RECOVER_OK,
                         RECOVER_FAIL };
-    typedef bool (*recoverFunc_type)(const std::string& strFile, std::string& out_backup_filename);
```

Comment:
Not sure what the state of this PR is, but I just saw this change: isn't it clearer to define a type here, instead of make the function type definition in-line in the parameter definition?